openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -40,7 +40,9 @@
                         <span class="fa fa-chevron-right ${ 'fa-rotate-90' if section_is_auto_opened else '' }" aria-hidden="true"></span>
                         <h3 class="section-title">${ section['display_name'] }</h3>
             % if section.get('complete'):
-                            <span class="complete-checkmark fa fa-check"></span>
+                            <span class="complete-checkmark fa fa-check">
+                                <span class="sr">${_("Completed")}</span>
```

Comment:
Could you please put the class="sr" spans outside the checkmark span, and add aria-hidden="true" to the checkmark span (as with the file above)?  As you have it now, the checkmark may be read by some screen readers.